### ISSUE TITLE
Code of Conduct

### ISSUE THREAD

**Issue by NataliaLKB** [node-girls/london#4]
Going through meeting minutes, this task was assigned to @msmichellegar.

Are you still happy to do this?

**Comment by iteles** [comment]
I think as a start we could use this one: http://confcodeofconduct.com/ (it's what a lot of meetings and conferences base theirs on).

**Comment by msmichellegar** [comment]
Yep! Was looking at a few different open source codes of conduct, but that looks good @iteles! I'll add a code of conduct repo in the next few days :)

**Comment by iteles** [comment]
🎉👌🏻
